### PR TITLE
Port interrupt fix to release branch

### PR DESCRIPTION
**Pull request by rchiodo**
For #8945

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -320,7 +343,7 @@ make this extension useful:
 
 ### Code Health
 
-1. Replace instances of ```traceInfoIf``` with ```traceInfoIfCI``` as almost all of them were just for CI tracing.
```
Comment (rchiodo) [inline CHANGELOG.md:323]: Crap something is formatting markdown on my machine

Comment (rchiodo) [reply CHANGELOG.md:323]: Oh this seems to be okay. It looks the same when renderered

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,8 +39,7 @@
    ([#8441](https://github.com/Microsoft/vscode-jupyter/issues/8441))
 1. Add support for pre-release version of Extensions for VS Code Insiders.
    ([#8463](https://github.com/Microsoft/vscode-jupyter/issues/8463))
-1. Allow to retrieve the jupyter filename programmatically (__vsc_ipynb_file__) [fix support for ClearML]
-   ([#8475](https://github.com/Microsoft/vscode-jupyter/issues/8475))
+1. Allow to retrieve the jupyter filename programmatically (**vsc_ipynb_file**) [fix support for ClearML](<[#8475](https://github.com/Microsoft/vscode-jupyter/issues/8475)>)
```
Comment (DonJayamanne) [inline CHANGELOG.md:42]: Why the angle brackets?

File: CHANGELOG.md [hunk 3]
```diff
@@ -635,7 +655,7 @@ make this extension useful:
    ([#6019](https://github.com/Microsoft/vscode-jupyter/issues/6019))
 1. Ship require.js with our notebook preloads and renderers.
    ([#6034](https://github.com/Microsoft/vscode-jupyter/issues/6034))
-1. Save output in *.ipynb even when output is created without any Jupyter output metadata.
+1. Save output in \*.ipynb even when output is created without any Jupyter output metadata.
```
Comment (DonJayamanne) [inline CHANGELOG.md:658]: Unnecessary `\`

File: CHANGELOG.md [hunk 4]
```diff
@@ -698,7 +718,7 @@ make this extension useful:
 
 ### Enhancements
 
-1. Data Viewer Filter Rows must use explicit wildcards to search for substrings in string filters. For example, filtering by "stable" will not show the value "unstable" anymore, but filtering by "*stable" will show "stable" and "unstable".
+1. Data Viewer Filter Rows must use explicit wildcards to search for substrings in string filters. For example, filtering by "stable" will not show the value "unstable" anymore, but filtering by "\*stable" will show "stable" and "unstable".
```
Comment (DonJayamanne) [inline CHANGELOG.md:721]: Unnecessary `\`

**Review by DonJayamanne — COMMENTED** [review]
Few issues with markdown, else all good.

File: CHANGELOG.md (hunk 2 above)
Comment (rchiodo) [reply CHANGELOG.md:42]: I didn't add any of these changes to the changelog. Some markdown auto editing thing did.

**Comment by rchiodo** [comment]
@DonJayamanne what do you think?

**Pull request merged**